src/test/java/WebDriverScenarioWithTests.java

```diff
@@ -0,0 +1,122 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import ru.yandex.qatools.ashot.AShot;
+import ru.yandex.qatools.ashot.Screenshot;
+import ru.yandex.qatools.ashot.shooting.ShootingStrategies;
+
+import javax.imageio.ImageIO;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+
+
+public class WebDriverScenarioWithTests {
+
+        private WebDriver driver;
+        Logger logger = Logger.getGlobal();
+
+        @BeforeMethod(alwaysRun = true)
+        public void browserSetup() {
+        driver = new ChromeDriver();
+    }
+
+        @Test(description = "JIRA binding can be here")
```

Comment:
description is not related to the test

Reply:
It was done for better remembering - deleted, thank you.
See In the following commit.